### PR TITLE
chore(dw): factor some stuff out, improve table empty state

### PR DESCRIPTION
**Pull request by gitdallas**
addresses feedback relating to: https://issues.redhat.com/browse/RHOAIENG-4746

## Description
some refactoring including moving components within `SharedStates` into their own files and moving layout type stuff like `<Card>` out of chart/table/state components and into their parent component.

added true empty state component for dw table when there are no workloads at all (not just none within a filter)
image

## How Has This Been Tested?
ran through dw pages and made sure everything looks right

## Test Impact
updated a test to reflect the empty table state change

## Request review criteria:
everything still works, dw table has empty state for no workloads

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)

If you have UI changes: 

- [x] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change (find relevant UX in the [SMEs](https://github.com/opendatahub-io/odh-dashboard/tree/main/docs/smes.md) section).

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

File: frontend/src/pages/distributedWorkloads/global/workloadStatus/DWWorkloadsTable.tsx [hunk 1]
```diff
@@ -1,34 +1,31 @@
...
   }
 
   if (!workloads.loaded) {
     return (
-      <Card isFullHeight>
-        <Bullseye style={{ minHeight: 150 }}>
-          <Spinner />
-        </Bullseye>
-      </Card>
+      <Bullseye style={{ minHeight: 150 }}>
```
Comment (mturley) [inline frontend/src/pages/distributedWorkloads/global/workloadStatus/DWWorkloadsTable.tsx:26]: I think you can use your new LoadingState component here

**Review by lucferbux — APPROVED** [review]
/lgtm

**Comment by lucferbux** [comment]
/override ci/prow/images ci/prow/odh-dashboard-pr-image-mirror

**Pull request merged**